pkg/resource/model/testdata/sdkapi/describe/success_primary_container.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
can we fix a filename template such that these have a similar name - `success_primary_container.json` and `describe_success.json`

Reply:
renamed to success_